roles/baseline/defaults/main.yml

```diff
@@ -25,12 +25,25 @@ METRICS_SERVER_CONFIG:
   apiService:
     create: true
 
+## Ingress-nginx - Defaults
+ingressVersions:
+  k8sMinorVersionCeiling:
+    value: 21
+    api:
+      chartVersion: 3.20.1
+      appVersion: 0.43.0
+  k8sMinorVersionFloor:
+    value: 22
+    api:
+      chartVersion: 4.0.13
+      appVersion: 1.1.0
+
```

Comment:
This is good, but wanted to bring up a point on supporting n versions based on differing kubernetes versions. What if at kubernetes v1.25 these is also a change needed? How will this be addressed? Not saying this has to be fixed here, just a general comment. Solving for the problem today might not provide the true flexibility needed in the future.